.github/workflows/publish-docker.yml

```diff
@@ -28,9 +70,9 @@ jobs:
           echo using "${DOCKER_IMAGE}:${SHA} as base image"
 
           mkdir -p build 
-          wget https://github.com/PureStake/moonbeam/releases/download/$VERSION/moonbeam -O build/moonbeam
-          wget https://github.com/PureStake/moonbeam/releases/download/$VERSION/moonbeam-skylake -O build/moonbeam-skylake
-          wget https://github.com/PureStake/moonbeam/releases/download/$VERSION/moonbeam-znver3 -O build/moonbeam-znver3
+          wget https://github.com/moonbeam-foundation/moonbeam/releases/download/$VERSION/moonbeam -O build/moonbeam
```

Comment:
so for now we are maintaining both I assume?

Reply:
you mean of url ? It doesn't really matter as the purestake is redirected to moonbeam-foundation

Reply:
oh no I meant pushing to both dockerhub repositories. I probably did not pick the right line to reference it